Assets/HotmaxEditorVR/loadPropsMenu.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+public class loadPropsMenu : MonoBehaviour
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+    private stateManager.editorModes _editorMode = stateManager.editorMode;
+    private bool _playerIsLocomoting = stateManager.playerIsLocomoting;
+
+    //--child gameObject refs
+    [SerializeField]
+    public GameObject propTiles;
```

Comment:
change to propTilesContainer - leave pluralized variables to just collection types - makes code more readable. Not to be super nitpicky, just want to point it out for future variable naming convention